gettsim/transfers/kindergeld.py

```diff
@@ -156,12 +107,10 @@ def kindergeld_anspruch_nach_stunden(
 
     Returns
     -------
-    BoolSeries indiciating kindergeld eligibility.
+    Boolean indiciating kindergeld eligibility.
     """
-    out = alter < kindergeld_params["höchstalter"]["ohne_bedingungen"]
-    out = out | (
-        (kindergeld_params["höchstalter"]["ohne_bedingungen"] <= alter)
-        & (alter <= kindergeld_params["höchstalter"]["mit_bedingungen"])
+    out = (alter < kindergeld_params["höchstalter"]["ohne_bedingungen"]) | (
```

Comment:
Why changed from `<=` to `<` ?, also line below ?  Former seems to make more sense. Pls always test these edge cases!

Reply:
> Why do you think the former makes more sense?

Because I would interpret "höchstalter" as "last age still eligible", not "first age ineligible". Given what you write, we should change to 24 in yaml then.

Reply:
> Because I would interpret "höchstalter" as "last age still eligible", not "first age ineligible".

Makes sense.

On the other hand, the numbers mentioned in the law are 18 and 25. I tend towards leaving the numbers as is and improve the parameter description.

Alternative would be to change numbers to 17 and 24 and adjust code accordingly.